runtime-modules/council/src/lib.rs

```diff
@@ -314,13 +315,13 @@ pub trait ReferendumConnection<T: Config> {
     fn increase_option_power(membership_id: &T::MemberId, amount: &VotePowerOf<T>);
 }
 
-decl_storage! {
+decl_storage! { generate_storage_info
     trait Store for Module<T: Config> as Council {
         /// Current council voting stage
         pub Stage get(fn stage) config(): CouncilStageUpdate<T::BlockNumber>;
 
         /// Current council members
-        pub CouncilMembers get(fn council_members) config(): Vec<CouncilMemberOf<T>>;
+        pub CouncilMembers get(fn council_members): WeakBoundedVec<CouncilMemberOf<T>, T::CouncilSize>;
```

Comment:
Unfortunately there is no `WeakBoundedBTreeSet`. The advantage of `WeakBoundedVec` is that it has an unfallable `froce_from` method, which can be used during `on_initialize` (more specifically in `elect_new_council` method). `BoundedBTreeSet` only has a fallable `try_from`, which returns `Result`, but since we cannot fail in `on_initialize`, we'd have to resort to something like `debug_assert` + `unwrap_or` perhaps...

Reply:
Should we make an issue for it to tackle this in future?

Reply:
> I see. Can we get rid of use of try_from? I assume we are we using it to convert from the Vec of results from the referendum pallet? If that means more refactoring in referendum pallet, I think we can live with this.

I think that would mean referendum's `IntermediateWinners` and council's `CouncilMembers` would need to be the same type (ie. have the same bound), so referendum's `MaxWinnerTargetCount` and council's `CouncilSize` would need to be unified. I'll have that in mind when investigating https://github.com/Joystream/joystream/issues/4268.